src/lib/ledgers/five-bells-ledger.js

```diff
@@ -146,18 +134,24 @@ FiveBellsLedger.prototype._autofund = function * () {
       connector: config.getIn(['server', 'base_uri']),
       password: this.credentials.password,
       fingerprint: this.credentials.fingerprint
-    },
-    cert: admin.cert,
-    ca: admin.ca,
-    key: admin.key
-  }, 'could not create account at ledger ' + this.id)
+    }
+  }, 'could not create account at ledger ' + this.id, admin)
 }
 
-function * request_retry (opts, error_msg) {
+function * request_retry (opts, error_msg, credentials) {
   let delay = backoffMin
   while (true) {
     try {
-      let res = yield request(opts)
+      let res = yield request(lodash.defaults(opts, {
+        auth: credentials && {
```

Comment:
Oops, you're right about user/pass/username/password.

When credentials is doesn't include the basic auth credentials (e.g. if its using client certs instead) an auth object of `{user: undefined, pass: undefined}` will still get passed to request.